model/src/main/scala/za/co/absa/atum/model/AtumContext.scala

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2021 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.atum.model
+
+case class AtumContext(
+    partitions: Partitioning,
+    measures: Set[Measure] = Set.empty
+)
```

Comment:
**Less important:**
Missing `additionalInfo`

Reply:
okay. Will be empty by default, as we agreed on